desc/compute/_curve.py

```diff
@@ -647,42 +728,25 @@ def _x_sss_FourierXYZCurve(params, transforms, profiles, data, **kwargs):
     description="Position vector along curve",
     dim=3,
     params=["X", "Y", "Z", "knots", "rotmat", "shift"],
-    transforms={"method": []},
+    transforms={
+        "intervals": [],
```

Comment:
I know I'm OCD, but if you delete this unnecessary comma this becomes 1 line instead of 3 and is more readable

Reply:
Same for the similar functions below